test/hw3/test_task1.py

```diff
@@ -0,0 +1,17 @@
+from unittest.mock import Mock, call
+
+from All_home_works.hw3.task1 import cache
+
+
+def test__cache_decorator():
```

Comment:
Please add a test where you check the results of the function calls.  you can use `operator.add` as base functions.

Reply:
I do not understand clearly what do you want from me. And why it should be a different test, it's almost the same as the previous test, isn't it?

Reply:
> it's almost the same as the previous test, isn't it?

Yes, it is.  I want you to test this "almost" part.  This small difference is one of the critical parts of the requirements: `Would give out cached value`